### PR TITLE
lesson01 

### PR DESCRIPTION
**Pull request by SvetaPiven**

### CODE REVIEW

File: lesson01/src/task03.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class task03 {
```
Comment (rakovets) [inline lesson01/src/task03.java:1]:
```suggestion
public class Task03 {
```

File: lesson01/src/task04.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+public class task04 {
```
Comment (rakovets) [inline lesson01/src/task04.java:1]:
```suggestion
public class Task04 {
```

**Review by rakovets — CHANGES_REQUESTED** [review]
need to fix

File: lesson01/src/task03.java (hunk 1 above)
Comment (SvetaPiven) [reply lesson01/src/task03.java]: Task1 fixed

File: lesson01/src/task04.java (hunk 2 above)
Comment (SvetaPiven) [reply lesson01/src/task04.java]: Task2 fixed

**Pull request merged**